Translations/German.csv

```diff
@@ -4592,3 +4592,13 @@ last.meal,"Letzte Mahlzeit",,
 breakdown.BreakdownFoodPoisoning,"Lebensmittelvergiftung",,
 breakdown.BreakdownFoodPoisoning.desc,"{0} leidet an einer Lebensmittelvergiftung!",,
 act.BreakdownFoodPoisoning,"Leidet an einer Lebensmittelvergiftung.",,
+research.ready.to.unlock,"Forschung abgeschlossen.",,"{0} is a number, it says that X research items are ready to unlock right now"							
```

Comment:
Bessere Übersetzung hier wäre "Forschungen bereit zum aktivieren" da man dazu die Gegenstände (Datoids) Verfügbar hat. Ich denke Außerdem, dass da ein kleiner Fehler in der englischen Version unterlaufen ist und noch ein "{0}" voran gehört.

Reply:
Ja ich weiß das die Englische Datei noch nicht fertig ist. Hatte nur nichts zu tun und bin schonmal angefangen ^^